src/Objs/Product/ProductComponent.tsx

```diff
@@ -206,24 +211,51 @@ const CartActionButton = connect(function CartActionButton({
     )
   }
 
-  if (isInCart(product)) {
+  const quantity = quantityInCart(product)
+  if (quantity) {
     return (
-      <button
-        className="btn btn-sm btn-primary"
-        onClick={() => {
-          removeFromCart(product)
-          toast.info(cartRemovedMessage)
-        }}
-      >
-        <i className="bi bi-x-lg"></i>
-        {cartRemoveLabel}
-      </button>
+      <>
+        <button
+          aria-label="-"
+          className="btn btn-sm btn-primary my-1"
+          onClick={async () => {
+            await subtractFromCart(product)
+            toast.info(cartRemovedMessage)
+          }}
+        >
+          <i className="bi bi-dash-lg px-0" />
+        </button>
+
+        <strong className="px-2">{quantity}</strong>
```

Comment:
It should also be possible to manually enter a number (e.g. if you sell screws).